client-ts/signalr-protocol-msgpack/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@aspnet/signalr-protocol-msgpack",
+  "version": "1.0.0-preview1-t000",
+  "description": "MsgPack Protocol support for ASP.NET Core SignalR",
+  "main": "./dist/cjs/index.js",
+  "browser": "./dist/browser/signalr-protocol-msgpack.js",
+  "module": "./dist/esm/index.js",
+  "typings": "./dist/esm/index.d.ts",
+  "umd_name": "signalR.protocols.msgpack",
+  "directories": {
+    "test": "spec"
+  },
+  "scripts": {
+    "clean": "node ../node_modules/rimraf/bin.js ./dist",
+    "build": "npm run clean && npm run build:esm && npm run build:cjs && npm run build:browser && npm run build:uglify",
+    "build:esm": "node ../node_modules/typescript/bin/tsc --project ./tsconfig.json --module es2015 --outDir ./dist/esm --target ES2015 -d",
+    "build:cjs": "node ../node_modules/typescript/bin/tsc --project ./tsconfig.json --module commonjs --outDir ./dist/cjs --target ES5",
```

Comment:
Have to build twice to get the right ESM and CJS builds. I looked at using babel to translate from ESM to CJS (rather than compiling the TS files again) but a) the perf is basically the same and b) it wasn't perfect, the helpers that TS injects get funky